### ISSUE TITLE
[BUG] "pub global activate fvm" does not upgrade to 1.3.7, but installs 1.3.6 

### ISSUE THREAD

**Issue by jsroest** [leoafarias/fvm#208]
When I run  fvm use 1.22.5, I get the notice to upgrade to 1.3.7:
```
johannesroest@WS-SW103 order_app_for_bunzl % fvm use 1.22.5         
Project now uses Flutter: 1.22.5

___________________________________________________


FVM Update Available 1.3.6 → 1.3.7 
Changelog: https://pub.dev/packages/fvm/changelog
Run pub global activate fvm to update

___________________________________________________

```
Then I run: pub global activate fvm
```
johannesroest@WS-SW103 order_app_for_bunzl % pub global activate fvm
Package fvm is currently active at version 1.3.6.
Resolving dependencies... (1.2s)
+ args 1.6.0
+ charcode 1.1.3
+ checked_yaml 1.0.4
+ cli_dialog 0.3.3 (0.4.0 available)
+ cli_util 0.2.0
+ collection 1.14.13
+ console 3.1.0
+ date_format 1.0.9
+ freezed_annotation 0.12.0
+ functional_data 0.3.0
+ fvm 1.3.6 (1.3.7 available)
+ http 0.12.2
+ http_parser 3.1.4
+ io 0.3.4
+ json2yaml 1.0.1 (1.0.1+1 available)
+ json_annotation 3.1.1
+ meta 1.2.4
+ path 1.7.0
+ pedantic 1.9.2
+ plain_optional 0.1.3
+ pretty_json 1.1.0
+ process_run 0.11.0+2 (0.11.2+8 available)
+ pub_api_client 1.0.9
+ pub_semver 1.4.4
+ pubspec_parse 0.1.7
+ pubspec_yaml 2.0.1 (2.0.1+1 available)
+ source_span 1.7.0
+ string_scanner 1.0.5
+ sum_types 0.2.1+2
+ term_glyph 1.1.0
+ typed_data 1.2.0
+ vector_math 2.0.8
+ version 1.2.0
+ yaml 2.2.1
Precompiling executables... (2.3s)
Precompiled fvm:main.
Installed executable fvm.
Activated fvm 1.3.6.
```
I did run 'pub cache repair', but that did not solve it.
I also tried to deactivate fvm and activate it again.

For now I'm stuck on 1.3.6

Edit: My colleague running Win10 has exactly the same issue

**Comment by cabustos** [comment]
equal to me

**Comment by leoafarias** [comment]
@jsroest this is weird. I just ran a test on my windows 10 machine and I could not replicate the error. Could you provide the dart version which you are running?

**Comment by jsroest** [comment]
@leoafarias 
dart --version gives:
Dart SDK version: 2.9.3 (stable) (Tue Sep 8 11:21:00 2020 +0200) on "macos_x64"
which dart gives:
/usr/local/bin/dart

If it helps you, we could setup a teams/slack call. Then I can give you access to my system so you can do some tests.
By the way, thanks for FVM, it's awesome.

**Comment by jsroest** [comment]
I'm happy to inform you that I did find the problem/solution.

I uninstalled dart 2.9.3 and then installed the latest version, 2.10.5:
```
brew uninstall dart
brew install dart
```

Then tried again to update fvm:
`pub global activate fvm`

It updated fine to 1.3.7.

This was on macOS, but I assume updating dart will also solve this on Win10 as the error was identical.

If you agree, this issue can be closed.
